hack/verify-internal-modules.sh

```diff
@@ -26,7 +26,7 @@ kube::util::ensure_clean_working_dir
 kube::golang::setup_env
 
 _tmpdir="$(kube::realpath "$(mktemp -d -t verify-internal-modules.XXXXXX)")"
-#kube::util::trap_add "rm -rf ${_tmpdir}" EXIT
+kube::util::trap_add "rm -rf ${_tmpdir:?}" EXIT
```

Comment:
YOu can remove the trap 5 lines below, then

Reply:
does git worktree not have any other internal git state we need to cleanup?